docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -30,6 +30,10 @@ endif::[]
 * Internet access to download the installation packages.
 * A working DNS server, and a localhost and server name that resolve properly.
 * A system user with administrative permissions (`sudo`) to perform installation.
+* SELinux policy permits binding to the ICMP service
++
+Note that failure to do so may generate an ICMP permission issue when starting OpenNMS.
+(See https://opennms.discourse.group/t/h29-wont-start-with-permission-error-to-open-icmp-socket/2387/1[H29+ won't start with permission error to open ICMP socket] on Discourse.)
```

Comment:
Is the content of that Discourse article something that should be copied into the docs to show users how to identify SELinux conflicts?

Reply:
That's a good idea. We could create a separate topic just under the requirements section to do that. Only issue would be maintaining it in two places, if there are ever any updates to the Discourse article.

Reply:
The discourse article could be updated to say "this info has been added to the docs" to direct people here for any other future updates

Reply:
It looks like we should use the "considerations with SELinux" section of the Discourse article and put it here. The rest is more describing the problem that they would see after installation and startup. Could create a troubleshooting section and that info to it.

Reply:
Have added that information. Too much? Too little?